client/src/components/ratingsReviews_components/AddPhotos.jsx

```diff
@@ -77,20 +66,39 @@ const AddPhotos = ({ submission, setSubmission }) => {
       </Modal.Footer>
     </Modal>
   );
+  const handlePhotoVerify = (event) => {
+    event.preventDefault();
+    if (photosObj.match(/\.(jpeg|jpg|gif|png)$/) != null) {
+      setModalShow(true);
+      setNotValidPhoto('');
+    } else {
+      setNotValidPhoto('The image URL is invalid or unable to be uploaded');
+    }
+  };
 
   return (
     <Form.Group>
-      <Form.Label>Upload Photos {photoCount}/5</Form.Label>
+      <Form.Label>
+        Upload Photos
+        {' '}
+        {photoCount}
+        /5
+      </Form.Label>
       { (photoCount < 5) ? (
         <InputGroup className="uploadPhotos">
           <Form.Group />
           <Form.Control
             type="url"
             placeholder="Enter photo URL"
-            value={photosObj.url}
+            value={photosObj}
             onChange={(e) => handleChange(e)}
           />
-          <Button onSubmit={(e) => { e.preventDefault(); setModalShow(true); }} variant="secondary" type="submit">Add Photos</Button>
+          <Button onClick={(e) => { handlePhotoVerify(e); }} variant="secondary" type="submit">Add Photos</Button>
```

Comment:
You should be able to just do onClick={handlePhotoVerify} since the event is already passed in automatically